nixos/doc/manual/development/releases.xml

```diff
@@ -0,0 +1,235 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        xmlns:xi="http://www.w3.org/2001/XInclude"
+        version="5.0"
+        xml:id="ch-releases">
+
+<title>Releases</title>
+
+<section xml:id="release-process">
+  <title>Release process</title>
+
+  <para>
+    Going through an example of releasing NixOS 15.09:
+  </para>
+
+  <section xml:id="one-month-before-the-release">
+    <title>One month before the release</title>
+    <itemizedlist spacing="compact">
+      <listitem>
+        <para>
+          send a warning email to nix-dev
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          talk to Eelco and others what should be part of the release
```

Comment:
Maybe add more details about how to get something into the release ? (set the milestone in github PRs / issues + the blocker tag for the really important matters that should be addressed before release). The “talk to Eelco and others” part seems hard to track for an external contributor